blocks/quote/quote.js

```diff
@@ -0,0 +1,11 @@
+export default function decorate(block) {
+  const quote = block.firstElementChild.firstElementChild;
+  const attribution = block.lastElementChild;
+
+  quote.classList.add('quote-body');
+  attribution.classList.add('attribution');
```

Comment:
this is a perfect place to use the BEM model for the class name. `quote__attribution` would be a good candidate :)